python_modules/dagster/dagster_tests/definitions_tests/test_time_window_partitions.py

```diff
@@ -673,7 +674,7 @@ def test_time_partitions_weekly_partitions(
             [
                 "2021-11-07-00:00",
                 "2021-11-07-01:00",
-                "2021-11-07-01:00",
+                "2021-11-07-01:00_POST_DST_TRANSITION",
```

Comment:
fun fact -- this test case perfectly illustrates what was wrong with the old implementation (get_partition_keys() returned a list with multiple copies of the same string)